tools/Makefile.unix

```diff
@@ -192,11 +192,18 @@ NUTTXNAME = nuttx
 BIN       = $(NUTTXNAME)$(EXEEXT)
 
 all: $(BIN)
-.PHONY: dirlinks context clean_context check_context config oldconfig menuconfig nconfig qconfig gconfig export subdir_clean clean subdir_distclean distclean apps_clean apps_distclean
+.PHONY: dirlinks context clean_context check_context config oldconfig menuconfig nconfig qconfig gconfig export subdir_clean clean subdir_distclean distclean apps_clean apps_distclean check_format
 ifeq ($(GIT_DIR),y)
```

Comment:
For all of these.. same changes should be applied to Makefile.win.  No is using Makefile.win now, but we don't want it to get too far out of sync.

Reply:
@patacongo - did you see where I asked you for help on the Makefile Side? Please do a PR to this branch.

Reply:
Since it depeands on the existence of a bash script, there is not simple way to implement this in Makefile.win  That would involve writing a check_code_style.bat which no one is going to do for a long time.  So we can forget that for now.

Reply:
@patacongo - 

All the code review feeback is in.  I am done for the day. Please send PR to this branch if you can help fix the Makefiles stuff. and script.

After all I did fix all you CS violations that are on master!  We are big on QPQ here!!!

Reply:
I don't know what QPQ is Quid Pro Quo?  I thought that is what Trump does.

We are bit on serving the needs of all end-users above anything else and we are big on code quality.

Reply:
No one is going to be sending any PRs.  No GIT-dependent code is allowed in the build system file unless is is properly conditioned on the existence of .git/